dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -56,6 +56,26 @@ const shouldInsertMerchHigh = (
 	return index === desiredPosition;
 };
 
+const isNavList = (collection: DCRCollectionType) => {
+	return (
+		collection.collectionType == 'nav/list' ||
+		collection.collectionType == 'nav/media-list'
+	);
+};
+
+const isToggleable = (
+	index: number,
+	collection: DCRCollectionType,
+	isNetworkFront: boolean,
+) => {
+	if (isNetworkFront) {
+		return (
+			collection.displayName.toLowerCase() != 'headlines' &&
+			!isNavList(collection)
+		);
+	} else return index != 0 && !isNavList(collection);
```

Comment:
Might be a bit odd to reason about the `isNavList` twice? Maybe we could have an early return if `isNavList`?

```
const isToggleable = (
	index: number,
	collection: DCRCollectionType,
	isNetworkFront: boolean,
) => {
	if (isNavList(collection)) return false;
	if (isNetworkFront) {
		return collection.displayName.toLowerCase() != 'headlines';
	}
	return index != 0;
};
```